.goreleaser.yml

```
@@ -84,6 +84,49 @@ dockers:
+      - "--label=org.opencontainers.image.licenses=Apache-2.0"
+    extra_files:
+      # For Docker, use the copied config.yaml (which is docker.yaml)
+      - config.yaml
+      - config/models.yaml
+      - config/profiles/
+      - logs/.gitkeep
+
+docker_manifests:
+  - name_template: ghcr.io/thushan/olla:latest
+    image_templates:
+        - "ghcr.io/thushan/olla:latest-amd64"
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)